hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/file/OMDirectoryCreateResponse.java

```diff
@@ -39,11 +40,17 @@
   public static final Logger LOG =
       LoggerFactory.getLogger(OMDirectoryCreateResponse.class);
   private OmKeyInfo dirKeyInfo;
+  private List<OmKeyInfo> parentKeyInfos;
+  private boolean createPrefix;
 
   public OMDirectoryCreateResponse(@Nullable OmKeyInfo dirKeyInfo,
-      @Nonnull OMResponse omResponse) {
+      @Nullable List<OmKeyInfo> parentKeyInfos,
+      @Nonnull OMResponse omResponse,
+      boolean createPrefix) {
```

Comment:
We can avoid passing the createPrefix param and use parentKeyInfos != null check only instead.

Reply:
the createPrefix represents a configuration parameter. whereas parentKeyInfos can actually be null.

yes, things will work as you suggested. I can remove the createPrefix argument. But given that it represents a config control, I would wish to retain it.

If you insist, then I will go ahead and remove it.

Reply:
Sure. I am ok with keeping the createPrefix param.